[19:00:43] [Client thread/ERROR]: Can't find the resource index file: assets\indexes\1.8.json
[19:00:43] [Client thread/INFO]: Logging in with username & password
[19:00:43] [Client thread/INFO]: Setting user: __DayTrip__
[19:00:43] [Client thread/INFO]: (Session ID is token:eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIwYTA1ZmMzZDA1ZTJmN2FiNWQ3MDViZmViNTNjNzRhOCIsInlnZ3QiOiI2ZTEyY2E1MTY3NWI0Mjg3OTRiNjdiZjBhMjlkMmZhMyIsInNwciI6IjIzMjBkNTVmZjE2OTQzNmU4M2Y0YjE4NWZkMjJlYzliIiwiaXNzIjoiWWdnZHJhc2lsLUF1dGgiLCJleHAiOjE2MzMzMTI4NDMsImlhdCI6MTYzMzE0MDA0M30.SPTL5bo5wVE1wv7i8hfHHyyDVj_Q0tYZYJyTWk8x0hQ:2320d55f-f169-436e-83f4-b185fd22ec9b)
[19:00:45] [Client thread/INFO]: LWJGL Version: 2.9.4
[19:00:46] [Client thread/INFO]: Reloading ResourceManager: Default
[19:00:46] [Sound Library Loader/INFO]: Starting up SoundSystem...
[19:00:46] [Thread-9/INFO]: Initializing LWJGL OpenAL
[19:00:46] [Thread-9/INFO]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[19:00:46] [Thread-9/INFO]: OpenAL initialized.
[19:00:47] [Sound Library Loader/INFO]: Sound engine started
[19:00:49] [Client thread/INFO]: Created: 512x512 textures-atlas
[19:00:55] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:music.menu
[19:01:13] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:music.menu
[19:01:26] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:music.menu
[19:01:34] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:01:35] [Client thread/INFO]: Connecting to mc.hypixel.net, 25565
[19:01:36] [Client thread/INFO]: [CHAT]                                      
[19:01:36] [Client thread/INFO]: [CHAT]                          
[19:01:38] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f §6joined the lobby!
[19:01:39] [Client thread/INFO]: 17
[19:01:39] [Client thread/INFO]: 17
[19:01:40] [Client thread/INFO]: [CHAT] [Sunrise Client]: Enabled hack Entity Info Display! (Keybind: Z)
[19:01:40] [Client thread/INFO]: true
[19:01:40] [Client thread/INFO]: 44
[19:01:40] [Client thread/INFO]: 44
[19:01:41] [Client thread/INFO]: 17
[19:01:41] [Client thread/INFO]: 17
[19:01:41] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:42] [Client thread/INFO]: 30
[19:01:42] [Client thread/INFO]: 30
[19:01:43] [Client thread/INFO]: 31
[19:01:43] [Client thread/INFO]: 31
[19:01:43] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:43] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:music.game
[19:01:45] [Client thread/INFO]: 32
[19:01:45] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:45] [Client thread/INFO]: 32
[19:01:46] [Client thread/INFO]: 30
[19:01:46] [Client thread/INFO]: 30
[19:01:47] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:portal.portal
[19:01:47] [Client thread/INFO]: 17
[19:01:47] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:48] [Client thread/INFO]: 17
[19:01:48] [Client thread/INFO]: [CHAT] §b[MVP§0+§b] Dxge§f §6joined the lobby!
[19:01:49] [Client thread/INFO]: 57
[19:01:50] [Client thread/INFO]: [CHAT] §b[MVP§2+§b] Toodalooo§f §6joined the lobby!
[19:01:51] [Client thread/INFO]: 57
[19:01:52] [Client thread/INFO]: [CHAT] §b[MVP§2+§b] ilypwn§f §6joined the lobby!
[19:01:53] [Client thread/INFO]: [CHAT] §b[MVP§e+§b] derpy_noob321§f §6joined the lobby!
[19:01:54] [Client thread/INFO]: 17
[19:01:54] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:54] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:54] [Client thread/INFO]: 17
[19:01:54] [Client thread/INFO]: 17
[19:01:55] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:55] [Client thread/INFO]: [CHAT] §b[MVP§2+§b] iSL1PP3D§f §6joined the lobby!
[19:01:55] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:55] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:56] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:56] [Client thread/INFO]: 17
[19:01:56] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:56] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:56] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:56] [Client thread/INFO]: 17
[19:01:56] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:56] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:57] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:57] [Client thread/INFO]: 17
[19:01:57] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:57] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:57] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:57] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:57] [Client thread/INFO]: 17
[19:01:57] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:58] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:01:58] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:58] [Client thread/INFO]: 17
[19:01:58] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.chicken.plop
[19:01:59] [Client thread/INFO]: 1
[19:02:05] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §b[MVP§2++§b] sirukusu§f §6joined the lobby! §a<§c<§b<
[19:02:20] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:21] [Client thread/INFO]: [CHAT] §b[MVP§a+§b] DuckyzEpic§f §6joined the lobby!
[19:02:21] [Client thread/INFO]: [CHAT] DuckyzEpic found a ✰✰✰✰✰ Mystery Box!
[19:02:21] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:21] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:22] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:22] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:22] [Client thread/INFO]: [CHAT] SnakesRPoggers found a ✰✰✰✰✰ Mystery Box!
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.say
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:28] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:29] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:30] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §6[MVP§8++§6] Scykic§f §6joined the lobby! §a<§c<§b<
[19:02:30] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:31] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:31] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:32] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:32] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:33] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:33] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:34] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.say
[19:02:34] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:39] [Client thread/INFO]: [CHAT] §b[MVP§f+§b] Roborg8§f §6joined the lobby!
[19:02:42] [Client thread/INFO]: [CHAT] §b[MVP§8+§b] WhyMaster§f §6joined the lobby!
[19:02:42] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.say
[19:02:44] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:02:44] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:02:44] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:step.stone
[19:02:47] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §6[MVP§6++§6] Ily_L§f §6joined the lobby! §a<§c<§b<
[19:02:48] [Client thread/INFO]: [CHAT] §b[MVP§c+§b] 0ies§f §6joined the lobby!
[19:02:49] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.say
[19:02:49] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] Pure_Kaiju§f §6joined the lobby!
[19:02:51] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:mob.sheep.step
[19:03:03] [Client thread/INFO]: [CHAT] §b[MVP§2+§b] Singuwu§f §6joined the lobby!
[19:03:03] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:03] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:03] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:03] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:13] [Client thread/INFO]: [CHAT] §b[MVP§e+§b] Hayzon1§f §6joined the lobby!
[19:03:13] [Client thread/INFO]: [CHAT] §b[MVP§4+§b] Girat1na§f §6joined the lobby!
[19:03:13] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:03:15] [Client thread/INFO]: 54
[19:03:17] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:17] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:18] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:18] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:20] [Client thread/INFO]: [CHAT] §7BrothersQui§7: PVP
[19:03:20] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:21] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] Merry_Magma_Cube§f §6joined the lobby!
[19:03:21] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:random.click
[19:03:23] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:03:23] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f: oki
[19:03:24] [Client thread/INFO]: [CHAT] §7BrothersQui§7: WITH HANDS
[19:03:27] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f: sumo
[19:03:29] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §6[MVP§6++§6] ___JR____§f §6joined the lobby! §a<§c<§b<
[19:03:29] [Client thread/INFO]: [CHAT]  §b>§c>§a>§r §6[MVP§c++§6] JazzyAZ03§f §6joined the lobby! §a<§c<§b<
[19:03:31] [Client thread/INFO]: [CHAT] §7BrothersQui§7: noo
[19:03:31] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:03:38] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f: yes
[19:03:39] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:03:39] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f: fist fight
[19:03:44] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f: correct?
[19:03:44] [Client thread/INFO]: [CHAT] §b[MVP§2+§b] Eaxle§f §6joined the lobby!
[19:03:47] [Client thread/INFO]: [CHAT] §7BrothersQui§7: bro
[19:03:50] [Client thread/INFO]: [CHAT] §b[MVP§d+§b] toemilkman§f: wat
[19:03:53] [Client thread/INFO]: [CHAT] §3§6✫ §3Classic Diamond II§r §b[MVP§2+§b] ppnon§f: ehehhe
[19:03:54] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:03:55] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:03:57] [Client thread/INFO]: 1
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeObjective(Scoreboard.java:218) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleScoreboardObjective(NetHandlerPlayClient.java:1718) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:64) ~[bin/:?]
	at net.minecraft.network.play.server.S3BPacketScoreboardObjective.processPacket(S3BPacketScoreboardObjective.java:10) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(FutureTask.java:122) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.get(FutureTask.java:192) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:20) [bin/:?]
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1189) [bin/:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:371) [bin/:?]
	at net.minecraft.client.main.Main.main(Main.java:109) [bin/:?]
	at Start.main(Start.java:9) [bin/:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.scoreboard.Scoreboard.removeTeam(Scoreboard.java:296) ~[bin/:?]
	at net.minecraft.client.network.NetHandlerPlayClient.handleTeams(NetHandlerPlayClient.java:1827) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:140) ~[bin/:?]
	at net.minecraft.network.play.server.S3EPacketTeams.processPacket(S3EPacketTeams.java:12) ~[bin/:?]
	at net.minecraft.network.PacketThreadUtil$1.run(PacketThreadUtil.java:15) ~[bin/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_282]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_282]
	at net.minecraft.util.Util.func_181617_a(Util.java:19) ~[bin/:?]
	... 4 more
[19:04:01] [Client thread/INFO]: [CHAT] You are AFK. Move around to return from AFK.
[19:04:02] [Client thread/INFO]: 1
[19:04:02] [Client thread/INFO]: 1
[19:04:03] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:04:05] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:04:07] [Client thread/WARN]: Unable to play unknown soundEvent: minecraft:gui.button.press
[19:04:07] [Client thread/INFO]: Stopping!
[19:04:07] [Client thread/INFO]: SoundSystem shutting down...
[19:04:07] [Client thread/WARN]: Author: Paul Lamb, www.paulscode.com
